lib/md5-hex.js

```diff
@@ -21,4 +33,6 @@ module.exports = function (input) {
   }
 
   return hash.digest('hex');
-};
+}
+
+module.exports = md5sum;
```

Comment:
This breaks the typescript check because the type information from the code comments doesn't get correctly associated with the exported function. Is this code style more important that having good type hinting?

Reply:
Nah, thats fine, thats a good reason. Side-note: We should totally (future PR) move this project to TS.

Reply:
> We should totally (future PR) move this project to TS.

💯 Yes please.